devenv/docker/blocks/jaeger/docker-compose.yaml

```diff
@@ -25,7 +25,7 @@
   promtail:
     image: grafana/promtail:master
     volumes:
-      - ./docker/blocks/loki/config.yaml:/etc/promtail/docker-config.yaml
+      - ./docker/blocks/loki-promtail/promtail-config.yaml:/etc/promtail/docker-config.yaml
```

Comment:
Noticed this pointed to a file that no longer exists so fixed it by re-using the config from the loki-promtail docker block